### ISSUE TITLE
Drag selection does not exclude elements

### ISSUE THREAD

**Issue by petko5kov** [jsplumb/jsplumb#828]
We are using jsPlumb community edition v2.6.9 and we found an issue with drag selection.
Use the following example: https://blog.jsplumbtoolkit.com/posts/multiple-element-drag.html
STR:
1) Select few modules to drag them all. They all moved together- OK.
2) Deselect only one from the group.
3) Move the deselected module. All modules are moving together - NOK.
Expected behavior: the deselected module should move separately.

**Comment by sporritt** [comment]
yeah its how it works right now. the selected boxes are in the current selection and get dragged whenever a drag happens. a better approach is with "posses", which were i think introduced after that post, but i am on my phone right now and cant easily get a whole lot of facts together. try lo9king through the docs for drag posses.

**Comment by petko5kov** [comment]
Thanks, I'll try it, but keep in mind that if some modules are selected and try to drag non-selected node (even not deselected, but just never selected), then the whole group is moving to.
It seems this group selection is not working very well.
